src/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/6495857eb4.js" crossorigin="anonymous"></script>
+    <title>Webpack Exercise</title>
+</head>
+<body>
+    <div class="container">
+        <div class="list-holder">
+            <div class="heading">
+                <p>Today's To Do</p>
+                <i class="fa-solid fa-rotate"></i>
+            </div><hr>
+            <input type="text" placeholder="Add to your list" id="description-input"><hr>
+        
+        </div>
+        
+        <button class="clear">Clear all completed</button>
```

Comment:
Check all your attributes carefully🥰. like you didn't give the type attribute to the button😊.